File: spring_java/src/main/java/dev/pollito/spring_java/sakila/film/adapter/in/rest/dto/FilmResponse.java
package dev.pollito.spring_java.sakila.film.adapter.in.rest.dto;

import static lombok.AccessLevel.*;

import lombok.*;
import lombok.experimental.FieldDefaults;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@FieldDefaults(level = PRIVATE)
public class FilmResponse {
  Long id;
  String title;
  String description;
  Integer releaseYear;
  String rating;
  Integer lengthMinutes;
  String language;
}

File: spring_kotlin/src/main/kotlin/dev/pollito/spring_kotlin/sakila/film/adapter/in/rest/dto/FilmResponse.kt
package dev.pollito.spring_kotlin.sakila.film.adapter.`in`.rest.dto

data class FilmResponse(
    val id: Long?,
    val title: String,
    val description: String,
    val releaseYear: Int,
    val rating: String,
    val lengthMinutes: Int,
    val language: String,
)
